src/components/platformFilter/client.tsx

```diff
@@ -47,11 +47,78 @@ export function PlatformFilterClient({platforms}: {platforms: Platform[]}) {
     })
   );
 
+  const mostViewedPlatforms: {icon: string; key: string; title: string; url: string}[] = [
```

Comment:
I would define this variable outside the component `PlatformFilterClient`. It's not dependent on any component logic and React would redeclare it on every render.